src/test/regress/expected/bfv_joins.out

```diff
@@ -4153,7 +4152,24 @@ from
     where
       ttt.tc1=ttt1.tc2
 ) t on true;
-ERROR:  Passing parameters across motion is not supported. (cdbmutate.c:3701)
+                               QUERY PLAN                               
+------------------------------------------------------------------------
+ Nested Loop Left Join
```

Comment:
Why did this case fixed, while next below not?

Reply:
Fixed the remaining test cases. Also added some more cases.

@RekGRpth , @bimboterminator1 , what do you think about merging test cases from 'goin_gp.sql' to 'dfv_joins.sql'?

Reply:
> merging test cases from 'goin_gp.sql' to 'dfv_joins.sql'?

Yes, of course you can move your tests there.

Reply:
Is it ok to add new cases in the middle of other patch tests?

Reply:
> Is it ok to add new cases in the middle of other patch tests?

I've placed new cases closer to similar existing queries. That seems logical. Anyway we are already changing the other patch.